PowerPointLabs/Test/UnitTest/PositionsLab/PositionsLabAlignTest.cs

```diff
@@ -433,6 +433,117 @@ public void TestAlignOneCenterDefault()
             CheckShapes(expectedShapes, actualShapes);
         }
 
+        [TestMethod]
+        [TestCategory("UT")]
+        public void TestAlignOneLeftSingleShape()
```

Comment:
What's the difference between this and the method `TestAlignOneLeftDefault`? Same with the other methods.

Reply:
The reference point is different. For the new tests, the reference points are the respective selected shapes.

Reply:
Thanks for the clarification, didn't spot that bit! :)